### ISSUE TITLE
Expected str, got int, crashes in HDF5 file writing 

### ISSUE THREAD

**Issue by prjemian** [BCDA-APS/apstools#466]
From https://github.com/APS-USAXS/ipython-usaxs/issues/459, this line:

```python
/home/beams11/USAXS/Documents/projects/apstools/apstools/filewriters.py in h5string(self, text)
   1125             return [self.h5string(t) for t in text]
```

**Comment by prjemian** [comment]
```
---: return [self.h5string(t) for t in text]
+++: return [self.h5string(str(t)) for t in text]
```

**Comment by prjemian** [comment]
This is already fixed in 9b79351 as part of #467: https://github.com/BCDA-APS/apstools/blob/5041417014e1473ff87434aee2d2246477596480/apstools/filewriters.py#L1125

This issue was not marked as fixed since a typo in the commit message placed it into (unrelated PR) #459.
